networkx/algorithms/tests/test_tournament.py

```diff
@@ -1,132 +1,157 @@
 """Unit tests for the :mod:`networkx.algorithms.tournament` module."""
 from itertools import combinations
-
-
+import pytest
 from networkx import DiGraph
 from networkx.algorithms.tournament import is_reachable
 from networkx.algorithms.tournament import is_strongly_connected
 from networkx.algorithms.tournament import is_tournament
 from networkx.algorithms.tournament import random_tournament
 from networkx.algorithms.tournament import hamiltonian_path
+from networkx.algorithms.tournament import score_sequence
+from networkx.algorithms.tournament import tournament_matrix
+from networkx.algorithms.tournament import index_satisfying
+
+np = pytest.importorskip("numpy")
+npt = pytest.importorskip("numpy.testing")
+
+
+def test_condition_not_satisfied():
+    condition = lambda x: x > 0
+    assert index_satisfying([0], condition) == 1
+
+
+def test_empty_iterable():
+    condition = lambda x: x > 0
+    with pytest.raises(ValueError):
+        index_satisfying([], condition)
+
+
+def test_is_tournament():
+    G = DiGraph()
+    G.add_edges_from([(0, 1), (1, 2), (2, 3), (3, 0), (1, 3), (0, 2)])
+    assert is_tournament(G)
+
+
+def test__loops():
+    """A tournament must have no -loops."""
```

Comment:
```suggestion
def test_self_loops():
    """A tournament must have no self-loops."""
```
Should this be `test_self_loops` or similar? Apologies if I'm misunderstanding some terminology.